.gitignore

```diff
@@ -40,7 +40,7 @@ jspm_packages
 lib
 
 # Package lock files
-package-lock.json
+yarn.lock
```

Comment:
We should track these

Reply:
Currently this repo uses NPM and this PR actually started tracking package-lock due to some auto upgrade issues that were happening. When we start using yarn, the exception for it should be removed to properly track the file.